adoc/chapters/programming_interface.adoc

```diff
@@ -11977,6 +11977,30 @@ size_t operator[](int dimension) const
    a@ Return the value of the specified dimension of the
       [code]#range#.
 
+a@
+[source]
+----
+size_t x() const noexcept;
```

Comment:
```suggestion
size_t x() const noexcept
```

There is no trailing semicolon for the entries in these tables.  Same below.

Reply:
Fixed in 2375fb06326a04f3a5183a5ae3947b6763406c3f.  I agree it makes sense to be consistent with the rest of the table.

This might be addressed by later passes to move everything to the new format, but I did notice that we're currently inconsistent.  These don't have semicolons and neither do some of the updated sections (e.g., the new math functions), but the group algorithms and some of the other new sections we're writing now do have semicolons.